### DISCUSSION TITLE
Vibrant Wallpapers

### DISCUSSION THREAD

**Discussion by itzsaltyz** [Atlas-OS/Atlas#13] [Community Artwork]
Hello. I made some wallpapers as some people(beta users) suggested :) There are not the best, cause i'm still new to Photoshop and etc.

![cod atlas os](https://user-images.githubusercontent.com/79692216/109280246-6fcfa600-7823-11eb-9c27-34307849da29.png)
![green atlasos](https://user-images.githubusercontent.com/79692216/109280256-72ca9680-7823-11eb-9428-3523b7d3716c.png)
![atlas os pink blue](https://user-images.githubusercontent.com/79692216/109280265-74945a00-7823-11eb-9c01-85426eeef57e.png)
![satisfactory atlas os](https://user-images.githubusercontent.com/79692216/109280269-75c58700-7823-11eb-82b2-b3d546262632.png)
![atlas os pink](https://user-images.githubusercontent.com/79692216/109280278-78c07780-7823-11eb-8996-a4eb55a1039b.png)
![atlasos fortnite](https://user-images.githubusercontent.com/79692216/109280281-7a8a3b00-7823-11eb-8cb7-e07aa89b0dfd.png)

**Comment by bevynfernandes** [comment]
Satisfactory one is epic!

**Reply by itzsaltyz** [reply]
true! 😄

**Comment by ketodin** [comment]
So clean! I love it. The only issue would be the third one, the logo does not seems centered. But anyway great work!

**Reply by itzsaltyz** [reply]
ahahah.. yeaah. i made this a whiile back and i completely forgot about my github XD might make some new ones, **if you have any ideas, let me know!**

**Comment by HanDong2810** [comment]
Great wallpaper！thumbs up for you！

**Reply by itzsaltyz** [reply]
**thank youu ❤️**
